server/crashmanager/templates/usersettings.html

```diff
@@ -39,6 +39,13 @@
 			<input type="hidden" name="changetemplate" value="1" />
 			<input type="submit" name="submit" value="Save Default Provider/Template" class="btn btn-danger"/>
 		</form>
+		{% if providers %}
```

Comment:
I would create a second panel named `Bug Providers settings`, and move that code in the new panel